### ISSUE TITLE
something wrong with migrate:rollback

### ISSUE THREAD

**Issue by supine-win** [thedevdojo/voyager#3157]
- Laravel Version: 5.5
- Voyager Version: 1.1
- PHP Version:7.1.6
- Database Driver & Version:5.7

### migrate:rollback wrong!
```bash
 SQLSTATE[HY000]: General error: 1832 Cannot change column 'role_id': used in a foreign key constraint 'users_role_id_foreign' (SQL: ALTER TABLE dg_users CHANGE role_id role_id INT DEFAULT NULL)
```
wrong in the Migration order?

**Comment by fletch3555** [comment]
Is this an upgrade or fresh install?

This issue has been reported before, but should've been fixed.

**Comment by reageek** [comment]
Experiencing the same issue.
There is a patch. It is waiting to be pulled in.
https://github.com/the-control-group/voyager/pull/3073

**Comment by emptynick** [comment]
Closing as #3073 was merged.

**Comment by tannerntannern** [comment]
It seems that this is still an issue... any updates?

**Comment by fletch3555** [comment]
The fix contained a breaking change, so it was merged into the 1.x branch.  It's fixed, but in a future 1.2 release, not 1.1.*

**Comment by priyag597** [comment]
https://github.com/the-control-group/voyager/issues/3071#issuecomment-421146772

voyager version : v1.1
laravel : v5.7

I made changes in seeders and re-seed the database 
run this command
 ### **php artisan migrate:fresh --seed** 
I am phasing this error 
### **Screenshot**
![image](https://user-images.githubusercontent.com/46341574/51971659-b49b7e00-249f-11e9-92ca-c6da8c6ad663.png)

Uploading 2017_11_26_013051_add_user_role_relationship_fix.php. this file is there in migration everything in commented in run() and down() method.

how to solve this please anyone can help me
